app/qml/map/Highlight.qml

```diff
@@ -72,6 +74,16 @@ Item {
   property real refTransformOffsetX: 0
   property real refTransformOffsetY: 0
 
+  SnapUtils {
```

Comment:
We might go with a different approach here. The thing is we want to reuse `Highlight` in different places and not all instances need to have the ability to snap.

I was thinking about having a `center` property here in `Highlight.qml` that would be binded by default to `Qt.point( highlight.width / 2, highlight.height / 2 )` and inside `MapWrapper` we can override this for `_highlightIdentified` with the logic you added to `crosshairPoint()` function.

That way we (a) do not need to have 2 `SnapUtils` objects, (b) can remove the `crosshair()` function altogether - just replace its one occurrence with the new property - and (c) `Highlight` can be reused 🙂

Reply:
Hmm, if I understand correctly, in that case we should move snapUtils to map wrapper and pass it as a property to both crosshair and highlight objects, right?